authz/providers/azure/rbac/checkaccessreqhelper.go

```diff
@@ -241,7 +440,12 @@ func getResultCacheKey(subRevReq *authzv1.SubjectAccessReviewSpec) string {
 	if subRevReq.ResourceAttributes != nil {
 		cacheKey = path.Join(cacheKey, defaultDir(subRevReq.ResourceAttributes.Namespace))
 		cacheKey = path.Join(cacheKey, defaultDir(subRevReq.ResourceAttributes.Group))
-		action := getResourceAndAction(subRevReq)
+		action := ""
+		if subRevReq.ResourceAttributes.Verb == "*" {
```

Comment:
should we handle * via getActionName itself?

Reply:
done